.github/workflows/pr-preview.yml

```diff
@@ -0,0 +1,72 @@
+name: 'ecma262'
+
+on: [pull_request_target]
+
+jobs:
+  check-membership:
+    name: 'check membership'
+    runs-on: ubuntu-22.04
+    outputs:
+      result: ${{ steps.test-membership.outputs.result }}
+    steps:
+      - uses: actions/github-script@v7
+        id: test-membership
+        env:
+          AUTHOR: ${{ github.event.pull_request.user.login }}
+        with:
+          result-encoding: string
+          script: |
+            const author = process.env.AUTHOR;
+            if (author === 'jmdyck') return 'yes';
```

Comment:
doesn't this mean that PRs from non-TC39 people won't get previews? that seems pretty limiting

Reply:
Yeah I mean it's not ideal but it's better than the current state of things. Ideally we'd also have some way to trigger it based on a label or comment on the PR but this is at least a start.

Reply:
so you have to remove the label and then re-add it to rebuilt the preview?

Reply:
The action removes the label automatically, but you do have to re-add it to rebuild, yes.

Could have it just work as long as the label is present but I figured if we're worried about malicious users requiring manual intervention for every change was the way to go.

Reply:
oh ok cool, that sounds great then!